app/serializers/activitypub/note_serializer.rb

```diff
@@ -13,12 +13,17 @@ class ActivityPub::NoteSerializer < ActiveModel::Serializer
   has_many :media_attachments, key: :attachment
   has_many :virtual_tags, key: :tag
 
+  has_many :poll_loaded_options, key: :one_of, if: :poll_and_not_multiple?
+  has_many :poll_loaded_options, key: :any_of, if: :poll_and_multiple?
+
+  attribute :closed, if: :poll_and_expires_at?
```

Comment:
Shouldn't this only be defined if the poll is *already closed*, not only if it has an expiration date set?

Reply:
We need to know in advance when it closes so users don't submit votes in vain. I think future date is fine, it is not a boolean after all

Reply:
It *can* be a boolean, and the spec says “Indicates that a question has been closed, and answers are no longer accepted.” https://www.w3.org/TR/activitystreams-vocabulary/#dfn-closed
It really seems weird to me to put a future date here :/

Reply:
Well, that sucks. How are we supposed to serialize this data then?

Reply:
`endTime`